metabase/metabase
# fix visible y-axis line

Closes https://github.com/metabase/metabase/issues/36821

### Description

We were displaying the y-axis line on some cases of the cartesian chart types with echarts. We should not have been doing this since in the previous implementation, we never displayed the y-axis line. This PR fixes this by adding the echarts options to hide the line.

### How to verify

1. Create any line, area, bar, or combo chart
2. Set x-axis scale to linear
3. Add the question to a dashboard
4. Send a subscription email with that dashboard
5. Confirm line doesn't appear in the subscription

### Demo

![after.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/9ULpbAzGBtvQRyZEpGvu/26584a35-5a22-42a0-ba39-6f14f7b8bc7c.png)

### Checklist
